commands/quiz.js

```diff
@@ -33,20 +50,42 @@ module.exports = {
             )
         });
 
-        await interaction.reply({ content: message, components: [answerButtons] });
+
 
         const collector = interaction.channel.createMessageComponentCollector({ time: 300000 });
 
         collector.on('collect', async interaction => {
-            if (interaction.customId == `answer-${answer}`) {
-                const answerOption = options[answer];
-                // TODO: figure out editReply vs reply
-                // interactionAlreadyAck'd
-                // https://stackoverflow.com/questions/71702161/interaction-has-already-been-acknowledged
-                await interaction.reply(`✅ <@${interaction.user.id}> is **CORRECT**! The right answer is ${answerLetter}: ${answerOption}`);
+            const answerOption = options[answer];
+
+            if (interaction.customId == `answer-reveal`) {
```

Comment:
We could have the answer reveal button be a configuration item in config.js, in case people don't want to enable/disable it.

```
"allow_answer_reveal:" false
```

_or_ we could also add an automatic reveal after a certain timeout! All just ideas for now.

Reply:
New update should be spicy, Boss!